entrypoint.sh

```diff
@@ -1,5 +1,14 @@
-#!/bin/sh
+#!/bin/bash
 
 set -e
 
-sh -c "vsce $*"
+if [ $1 == "yes" ]; then
+    vsce package
+    filelist=(`ls *.vsix`)
```

Comment:
this looks a bit fragile, just in case there are some committed .vsix files in the repo (there shouldn't be, but who knows...)

Maybe something like "most recently modified vsix file" would be more robust? 

```
file = $(realpath $(ls -t *.vsix | head -1))
```
(not actually tested)

Reply:
Fixed

Reply:
I am not an expert in Bash scription. Thank you for correction.